modules-js/react-fleet/src/form-elements/inputs/MemorableDateInput.tsx

```diff
@@ -511,10 +511,10 @@ const FIELDS_CONTAINER_STYLING = css({
     },
   },
   input: {
-    '-moz-appearance': 'textfield',
+    MozAppearance: 'textfield',
```

Comment:
Whoa, didn’t realize vendor prefixes were available in Pascal case here.

Reply:
Yah, there’s a warning now that kebab-case isn’t supported in objects.